src/pubsub.js

```diff
@@ -69,6 +69,22 @@
         subscriber( message, data );
     }
 
+    function findWildCardMessage(topic)
+    {
+        var foundMessages = Object.keys(messages).find(function(messageItem){
+            return messageItem.indexOf('*') > -1 && matchMessage(topic,messageItem);
+        });
+        return foundMessages || []; 
+    }
+
+    function matchMessage(str, rule) {
+        var escapeRegex = (str) => str.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, "\\$1");
```

Comment:
This codebase should by syntactically compatible with ES3, so I am afraid that means no arrow functions.

> ES3 compatible. PubSubJS should be able to run everywhere that can execute JavaScript. Browsers, servers, ebook readers, old phones, game consoles.

It does state this in the `README.md`, but not yet in `CONTRIBUTING.md`.